apps/web/playwright/managed-event-types.e2e.ts

```diff
@@ -207,7 +207,7 @@ test.describe("Managed Event Types", () => {
   });
 
   const MANAGED_EVENT_TABS: { slug: string; locator: (page: Page) => Locator | Promise<Locator> }[] = [
-    { slug: "setup", locator: (page) => getByKey(page, "title") },
+    { slug: "setup", locator: (page) => getByKey(page, "translate_description_button") },
```

Comment:
- if we look for "title" in event setup page, we have 2 instances (because now we have `Translate title/description to the visitor's browser language using AI` button).